src/shell/shell_cmds.cpp

```diff
@@ -2532,6 +2492,98 @@ static std::vector<std::string> search_files(const std::string_view query)
 	return files;
 }
 
+struct FileExt {
+	std::string root      = {};
+	std::string extension = {};
```

Comment:
Suggest going with `name` and `ext` instead. "root" has other connotations, and "name" + "extension" is the standard MS-DOS nomenclature.

Reply:
Also, maybe call this `DosFilename` or something and move it into some common DOS command / FAT / filesystem related header? (I don't deal with DOS filesystem stuff often, so you'll need to research this a bit 😄 )

Reply:
Renamed and moved to `dos_system.h`.  Could not find a pre-existing struct for this.  Closest I saw was `DOS_File` but that's a heavy weight class with a bunch of virtual functions and such.